packages/astro/src/default/components/webcontainer.ts

```diff
@@ -1,14 +1,16 @@
-import { useAuth } from './setup.js';
-import { auth, WebContainer } from '@webcontainer/api';
 import { TutorialStore } from '@tutorialkit/runtime';
+import { auth, WebContainer } from '@webcontainer/api';
+import { useAuth } from './setup.js';
```

Comment:
Oh this import should always be first as it sets a global read by the `@webcontainer/api` package.